scripts/parse_release.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/**
+ * @file
+ * Returns the latest released tag for given module.
+ */
+
+$shortopts = "m:";
+// Required value.
+$shortopts = "a";
+
+$longopts = [
+  "module:",
+  "all",
+];
+$options = getopt($shortopts, $longopts);
+$module = $options['module'] ?? "";
+$all = isset($options['a']);
+if (!$module && !$all) {
+  log_message("The --module option is required option or use -a to get release information for all modules.", "error");
+  exit;
+}
+$modules = [
+  "acquia_cms_article" => "1.x",
+  "acquia_cms_audio" => "1.x",
+  "acquia_cms_common" => "2.x",
```

Comment:
Will it automatically do release for common 1.x

Reply:
No, it won't.